test/toolbox_image.sh

```diff
@@ -185,7 +184,7 @@ test_update_files_command()
 
 test_jedi_exclude_list_exists()
 {
-    test -f "${toolbox_root_dir}/${JEDI_EXCLUDE_LIST_FILE}" || return 1
+    test -d "${toolbox_root_dir}/${SYSTEM_UPDATE_CONF_DIR}" || return 1
```

Comment:
This test just doesn't make that much sense now since it is not an input anymore and we do validate that the filter works correctly. But we never check that the filter 'filters' what we explicitly wan't to filter.